src/Tribe/Settings_Helper.php

```diff
@@ -5,228 +5,222 @@
 	die( '-1' );
 }
 
-if ( class_exists( 'Tribe__Extension__Settings_Helper' ) ) {
-	return;
-}
-
-/**
- * Helper for inserting/removing fields on the WP Admin Tribe Settings pages
- */
-class Tribe__Extension__Settings_Helper {
-
-	/**
-	 * Fields inserted into misc section
-	 *
-	 * @var array
-	 */
-	private $insert_fields_misc = array();
-
-	/**
-	 * Fields that will be inserted above a specified field
-	 *
-	 * @var array
-	 */
-	private $insert_fields_above = array();
-
-	/**
-	 * Fields that will be inserted below a specified field
-	 *
-	 * @var array
-	 */
-	private $insert_fields_below = array();
-
-	/**
-	 * Array of settings being added to a Tribe Settings tab
-	 *
-	 * @var array
-	 */
-	private $remove_fields = array();
-
-
+if ( ! class_exists( 'Tribe__Extension__Settings_Helper' ) ) {
 	/**
-	 * Setup the helper
-	 *
-	 * @param int $priority Priority at which this hooks into 'tribe_settings_tab_fields'.
+	 * Helper for inserting/removing fields on the WP Admin Tribe Settings pages
 	 */
-	public function __construct( $priority = 100 ) {
-		add_filter( 'tribe_settings_tab_fields', array( $this, 'filter_options' ), $priority, 2 );
-	}
+	class Tribe__Extension__Settings_Helper {

```

Comment:
I _think_ a lot of the changes here are formatting fixes. Can you summarize, though - did any methods change or were any method arguments altered etc?

Reply:
No substantive Settings Helper changes... just wrapped the whole thing in class_exists() and removed some extra line breaks -- so you're correct

Reply:
:+1: